src/Site/Map/SiteMap.tsx

```diff
@@ -37,9 +33,12 @@ export const SiteMapTable: FC = () => {
 
   useEffect(() => {
     if (!created) return setCreated(true);
+
     requery();
   }, [created, filter]);
 
+  useEffect(() => requery(), [perPage, currentPage]);
```

Comment:
You have double the code, Both useEffects do the same thing just different dependancies

Reply:
done.